samples/radio_test/src/radio_test.h

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef RADIO_TEST_H_
+#define RADIO_TEST_H_
+
+#include <zephyr/types.h>
+#include <nrfx/hal/nrf_radio.h>
+
+/** Indicates devices that support BLE LR and 802.15.4 radio modes. */
+#define USE_MORE_RADIO_MODES (defined(NRF52840_XXAA) || \
+			      defined(NRF52833_XXAA) || \
```

Comment:
nrf52811 supports LR/15.4 etc.

Reply:
Yes, thanks for pointing it out, but I would like to stick only to targets that are built with CI (nrf52832, nrf52840, nrf5340).